Package.swift

```diff
@@ -8,13 +8,6 @@ let package = Package(
     .plugin(name: "FormatSwift", targets: ["FormatSwift"]),
   ],
   dependencies: [
-    .package(url: "https://github.com/calda/SwiftFormat", exact: "0.49.11-beta-2"),
-    // The `SwiftLintFramework` target uses "unsafe build flags" so Xcode doesn't
-    // allow us to reference a specific version number. To work around that,
-    // we can reference the specific commit for that version.
-    //  - This is top-of-tree master from 7/22/22, because the most recent release version
-    //    (0.47.1) seems to have issues in some real-world projects like airbnb/epoxy-ios.
-    .package(url: "https://github.com/realm/SwiftLint", revision: "c5aa806"),
```

Comment:
Will this allow us to not have the full source of these two tools and all of the deps in projects that consume this plugin? I was noticing that Epoxy had [way more resolved package deps](https://github.com/airbnb/epoxy-ios/blob/9b9b1d3312dc2daaaeb06346b72451e8accfe7dd/Epoxy.xcworkspace/xcshareddata/swiftpm/Package.resolved#L40-L120) after we added this dep

Reply:
Yes! The only remaining source dependency is Swift Argument Parser, everything else is precompiled.